CorsaSchoolManagementSystem/Models/StudentFees.php

```diff
@@ -0,0 +1,16 @@
+<?php 
+
+require_once("Core/init.php");
```

Comment:
How does the StudentFees Class depend on the init file?

Reply:
This class won't produce any error or exception which requires that file. The only error likely to have would be syntax error and the syntax is correct so would be recommended to remove that line of code.

Reply:
Also make sure if you resolve any issue mentioned via comment by clicking on resolve conversation button

Reply:
Aright pls